CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [4.13.0](https://github.com/tv2/sisyfos-audio-controller/compare/v4.12.0...v4.13.0) (2021-11-09)

```

Comment:
Heads up: In future versions we won't be using standard-version, since a push to master will trigger server bumping and changelog generation.
The pipeline for changelog might need som extra attention, since currently it just prepends the changes to existing changelog file.

Reply:
👍